h2o-bindings/bin/custom/python/gen_targetencoder.py

```diff
@@ -15,8 +15,22 @@ def transform(self, frame, data_leakage_handling="None", noise=-1, seed=-1):
         :param int seed: a random seed used to generate draws from the uniform distribution for random noise. Defaults to -1.
 
         :example:
-        >>> targetEncoder = TargetEncoder(encoded_columns=te_columns, target_column=responseColumnName, blended_avg=True, inflection_point=10, smoothing=20)
-                            >>> encodedTrain = targetEncoder.transform(frame=trainFrame, data_leakage_handling="None", seed=1234, is_train_or_valid=True)
+        >>> titanic = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv")
+        >>> predictors = ["home.dest", "cabin", "embarked"]
+        >>> response = "survived"
+        >>> titanic["survived"] = titanic["survived"].asfactor()
+        >>> fold_col = "kfold_column"
+        >>> titanic[fold_col] = titanic.kfold_column(n_folds=5, seed=1234)
+        >>> titanic_te = H2OTargetEncoderEstimator(k = 0.7,
+        ...                                        f = 0.3,
+        ...                                        data_leakage_handling = "None",
```

Comment:
in the examples, please avoid specifying parameters if you're using their default value: here `data_leakage_handling`. This applies to many of the examples below.

This param has a flaw: it provides 2 representations for the same value, `data_leakage_handling=None` (default) and `data_leakage_handling="None" which can only create confusion, so let's stick with the usual None value.